public/assets/js/accordions.js

```diff
@@ -12,20 +12,57 @@
   */
 
 export function Accordions () {
-  const accordion = document.querySelector('.accordion')
-  if (accordion) {
-    const accordionItems = document.querySelectorAll('.accordion__item')
-    accordionItems.forEach((accordionItem) => {
-      accordionItem.addEventListener('click', (e) => {
-        const expanded = accordionItem.getAttribute('aria-expanded') === 'true' || false
-        accordionItem.setAttribute('aria-expanded', !expanded)
+  const accordions = document.querySelectorAll('.accordion-group')
 
-        // close all other accordion items... maybe?
-        // accordionItems.forEach((accordionItem) => {
-        //   if (accordionItem !== e.currentTarget) {
-        //     accordionItem.setAttribute('aria-expanded', false)
-        //   }
-        // })
+  if (accordions) {
```

Comment:
We should check here if there's no accordions we should just return the function, is faster and self explanatory.

What you did is fine, is just faster/better do an early return if there's no elements to be manipulated.


```suggestion
  if (!accordions) {
    return;
  }
```